docs/style-guide.rst

```diff
@@ -2,50 +2,12 @@
 Hy Style Guide
 ==============
 
-   “You know, Minister, I disagree with Dumbledore on many counts…but
-   you cannot deny he’s got style…”
-   — Phineas Nigellus Black, *Harry Potter and the Order of the Phoenix*
-
 The Hy style guide intends to be a set of ground rules for the Hyve
 (yes, the Hy community prides itself in appending Hy to everything)
 to write idiomatic Hy code. Hy derives a lot from Clojure & Common
 Lisp, while always maintaining Python interoperability.
 
 
-Prelude
-=======
-
-The Tao of Hy
--------------
-
-.. code-block:: none
-
-   Ummon asked the head monk, "What sutra are you lecturing on?"
-   "The Nirvana Sutra."
-   "The Nirvana Sutra has the Four Virtues, hasn't it?"
-   "It has."
-   Ummon asked, picking up a cup, "How many virtues has this?"
-   "None at all," said the monk.
-   "But ancient people said it had, didn't they?" said Ummon.
-   "What do you think of what they said?"
-   Ummon struck the cup and asked, "You understand?"
-   "No," said the monk.
-   "Then," said Ummon, "You'd better go on with your lectures on the sutra."
-   — the (koan) macro
-
-The following illustrates a brief list of design decisions that went
```

Comment:
Why remove this? From what I can tell it's something that's largely still followed, especially the `We're Homoiconic Python.`

Reply:
I don't feel that "largely" is good enough for reference documentation. For example, "Use Unicode everywhere" is out of date because it refers to our features to close the gap between Pythons 2 and 3 that are now gone. And it's not true that we always defer to Python before other Lisps. Sometimes we imitate Python and sometimes we imitate other Lisps. And "we're Homoiconic Python, with extra bits that make sense" doesn't do justice to all of the language's semantic additions. When Hy is discussed elsewhere, I see a lot of grousing by non-users that Hy isn't "really" Lisp, and while I've never gotten an answer as to why anybody thinks that, I suspect that outsiders assume that Hy does less than it really does.

Reply:
[For example.](https://www.reddit.com/r/lisp/comments/6j9mus/hy_0130_released/djddh0h/)